pom.xml

```diff
@@ -0,0 +1,208 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>Services</groupId>
+  <artifactId>ServicesUtilities</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>ServicesUtilities</name>
+  <description>Simple utilities for UCLA Services Team projects</description>
```

Comment:
I think we should specify "UCLA Library" here instead of "UCLA"

Reply:
Typo with the new version: UCLA LIbrary